libs/api-implementation/orchestration/src/lib/orchestration.controller.ts

```diff
@@ -12,7 +12,6 @@ export class VMOrchestrationController {
       await this.orchestrationService.createMachine(createMachineDTO.name);
       return `Machine ${createMachineDTO.name} successfuly created`;
     } catch (error) {
-      console.debug(error.stack);
       return `Error while creating the machine`;
```

Comment:
can you please elaborate on the change here?

Reply:
As I have already mentioned i have the following structure:
controller -> service
In service I have the structure:
`console.error()` by following `console.debug()` and rethrowing it to the controller. 
In the controller, I only have a generic message explaining what's going on. The reason why did I remove it, because it will duplicate the stack trace of the error.

Reply:
Yeah, I believe your reasoning makes more sense. Thanks for the time to explain.